sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -258,20 +260,21 @@ export class EducationProgramOfferingController extends BaseController {
   @AllowAuthorizedParty(AuthorizedParties.institution)
   @HasLocationAccess("locationId")
   @Get(
-    "location/:locationId/education-program/:programId/program-year/:programYearId/offerings-list",
+    "location/:locationId/education-program/:programId/program-year/:programYearId/selectedOffering/:selectedOffering/offerings-list",
```

Comment:
Also for the API endpoint, I would recommend using offeringIntensity in URI instead of selectedOffering.

Selected offering sounds like client specific language. APIs purpose must be independent of client and be more generic.